test/controllers/admin_controller_test.rb

```diff
@@ -7,12 +7,12 @@ class AdminControllerTest < ActionController::TestCase
 
   test 'should get show' do
     Rails.application.config.action_mailer.smtp_settings = {
-      address: APP_CONFIG['mailer_address'],
-      port: APP_CONFIG['mailer_port'],
-      domain: APP_CONFIG['mailer_domain'],
-      user_name: APP_CONFIG['mailer_user_name'],
-      password: APP_CONFIG['mailer_password'],
-      authentication: APP_CONFIG['mailer_authentication']
+      address: Cypress::AppConfig['mailer_address'],
```

Comment:
Correct, setting values is the only time there is a problem. The actual issue stems from when the tests write to cypress.yml.